### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by GD-Dheer**
Contributing Guide. Check and modify if there is anything missing. 

This pull request is an example of one that will only merged following at least 2 other developers approval.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- All development will take place on develop branch, this branch is created under the master branch. To update your local git to reflect remote git please use the following command inside your locally cloned repo.
+
+    git fetch --all
+    git checkout develop 
+
+    All feature branches are required to fork off develop branch and upon completion of your feature, you must submit a pull request from your feature branch to merge into develop. To create a branch under develop:
+
+    git checkout -b myfeature_branch develop
+
+- Feature branches should be appended with github issue associated to it. For example, lets say you  are working on a github issue referenced as '[Backend] Add Logic for Sign up #63'. To you should name your branch as 'Backend-SignUp-63'.
```
Comment (ds161833) [inline Contributing.md]: I suggest we put number upfront, eg 63-Backend-SignUp, nubmers will be easier to map in mind

Comment (GD-Dheer) [reply Contributing.md]: > I suggest we put number upfront, eg 63-Backend-SignUp, nubmers will be easier to map in mind

Yeah thats okay. Ill make the modification shortly

**Review by Mehrdadmaskull — APPROVED** [review]

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    git checkout -b myfeature_branch develop
+
+- Feature branches should be appended with github issue associated to it. For example, lets say you  are working on a github issue referenced as '[Backend] Add Logic for Sign up #63'. To you should name your branch as '63-Backend-SignUp'.
+
+- Git committing style. The command used to commit changes is:
+
+    git commit -m 'Message associated to your commit' 
+
+    When making commits you should include the issue number in the commit, this allows the commits to be linked the git issue. For example if I was working under my branch 'Backend-SignUp-63' and a validator was implemented as part of the feature. The commit would look like '[#63] Add Validator for Input Fields'.
```
Comment (TiraFire) [inline Contributing.md]: Just a small change to the '63-Backend-Signup' to be the same as above

**Review by TiraFire — CHANGES_REQUESTED** [review]
Looks good, just fix the small error in consistency ^^

**Pull request merged**